src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -100,7 +100,10 @@ ClosureRemap build_remap(const Function& fbody, const DCOFFParams& params_to) {
             // FIXME: type should be queried from a lazy tensor
             // and compared against param->get_element_type()
             // to decide 100%
-            m.weights_to_unpack.insert(i - fbody._param_offset);
```

Comment:
let's finally introduce a temporary variable as this `i - offset` is seen three times in this poor `else if`

Reply:
Done